devel/ninja/Makefile

```diff
@@ -0,0 +1,77 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=ninja
+PKG_VERSION:=1.9.0
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/ninja-build/ninja/tar.gz/v$(PKG_VERSION)?
```

Comment:
codeload should not be used directly. The normal release tarball listed under releases is preferred.

Reply:
Ok, I switched meson to that, but the ninja releases are different it seems. The link on the github releases redirects to codeload (`curl "https://github.com/ninja-build/ninja/archive/v1.9.0.tar.gz"|xxd`) and that fails to download, so I kept codeload there.

Reply:
That seems bizarre. Note that https://github.com/ninja-build/ninja/archive/v1.9.0/ninja-1.9.0.tar.gz works. Well, it redirects to that codeload link. I'd just leave it.